serialzy/serializers/onnx.py

```diff
@@ -0,0 +1,54 @@
+import inspect
+import os
+from pathlib import Path
+from typing import BinaryIO, Union, Type, Optional, Any
+
+from serialzy.serializers.base_model import (
+    ModelBaseSerializer,
+    deserialize_from_file,
+    serialize_to_file,
+    unpack_model_file
+)
+
+
+class ONNXSerializer(ModelBaseSerializer):
+    def __init__(self):
+        super().__init__("onnx", __name__)
```

Comment:
The first argument is "module_name", second - "serializer_name".
So why are you passing `__name__` (which is module name) to `serializer_name`?)))

Reply:
because it's used as arg in `logging.getLogger` call